lib/src/empty_unmodifiable_set.dart

```diff
@@ -6,15 +6,10 @@ import 'dart:collection';
 
 import 'unmodifiable_wrappers.dart';
 
-// Unfortunately, we can't use UnmodifiableSetMixin here, since const classes
-// can't use mixins.
 /// An unmodifiable, empty set that can have a const constructor.
 class EmptyUnmodifiableSet<E> extends IterableBase<E>
+    with UnmodifiableSetMixin<E>
```

Comment:
This seems like a good change, but I don't understand how it's related. Can this go out in a separate commit?

Reply:
It's an unrelated drive-by fix. I can do that by itself too.

Reply:
> I can do that by itself too.

I think that would be better. The travis wait time for individual PRs is the only downside - if you need to send a few things together to work around that I think it's OK, otherwise for isolated refactorings I think we should follow go/small-cls